Add cleanup security scans

m1kra opened the pull request:
Transition action "cleanup security scans" should be available for VirtualServer and CloudHost, RAS-1215.

m1kra committed:
add tests

m1kra committed:
add cleaning security scans RAS-1215

mkurek commented on src/ralph/virtual/models.py:
```diff
@@ -267,6 +268,18 @@ def ip_addresses(self, value):
+    @transition_action(
+        verbose_name=_('Cleanup security scans'),
+    )
+    def cleanup_security_scans(cls, instances, **kwargs):
```
the same method is defined already for `DataCenterAsset`. Please extract it to some common class or put somewhere as plain function - `transition_action` has additional argument - `models` - see https://github.com/allegro/ralph/blob/ng/src/ralph/deployment/deployment.py for examples.

vi4m commented:
@m1kra description is required for all pull requests (can re-paste from RAS-1215)

m1kra committed:
remove cleanup_security_scans from models

m1kra committed:
add transition actions in security

m1kra commented:
@vi4m Thank you, I have added the description.

@mkurek I have moved `cleanup_security_scans` to a new file in security app and added config loading it in `AppConfig.ready()`, following the approach in the deployment module. 
I am not quite sure if security module is the best location for it, but I find this solution logical. I was also thinking of adding it to transition app or base assets, so the additional config wouldn't be necessary.

ar4s commented on src/ralph/security/apps.py:
```diff
@@ -0,0 +1,16 @@
+
+    def get_load_modules_when_ready(self):
+        """
+        security/transitions.py will be loaded alongside modules reurned by
```
reurned -> returned

m1kra replied:
Done.

m1kra committed:
correct spelling
